### PR TITLE
LPS-192285 Draggable area active just when item from the accordions is selected

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
The motivation behind this ticket is to improve Segments Editor usability and accessibility by adding different styles in the drag and drop process.
https://liferay.atlassian.net/browse/LPS-192285

Proposed Solution
========
What I did to solve this problem is to use the canDrop variable to know if an element is being dragged to change the style of the dropTarget.

Steps to Verify
========
1. Go to Product Menu -> People -> Segments -> Create a new Segment and save it
2. Edit the segment and start to add a new property from a different Contributor. (e.g if the created segment contains a property of type Session, this one must be Organization, User or Segment type)
3. Start dragging the one property to the dropTarget and check that:
   - When the new Contributors panel is expanded, the border of the drop target is default (grey) and dashed
   - When an element is being dragged the border turns to blue and dashed
   - When the element is over the drop target it turns blue and solid

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by Stephy6** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
@Stephy6 I think we should add `border-secondary` class when not adding `border-primary`, so the color is grey and not black

**Comment by Stephy6** [comment]
> @Stephy6 I think we should add `border-secondary` class when not adding `border-primary`, so the color is grey and not black

Thats true!

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/EmptyDropZone.js [hunk 1]
```diff
@@ -43,10 +43,17 @@ function EmptyDropZone({canDrop, connectDropTarget, emptyContributors, hover}) {
...
-				'empty-drop-zone-dashed border-primary rounded':
-					displayEmptyDropZone && (!canDrop || !hover),
-			})}
+			className={classNames(
+				'empty-drop-zone-root',
+				{
+					'empty-drop-zone-dashed rounded border-secondary':
+						displayEmptyDropZone && !canDrop,
+				},
+				{
```
Comment (sandrodw3) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/EmptyDropZone.js:52]: Hey @Stephy6, you don't need to put two different objects, you can use only one with all classes and conditions.

**Comment by sandrodw3** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/13190

**Pull request closed without merging**